### DISCUSSION TITLE
Trying to run on Red Hat Enterprise Linux release 8.6 (Ootpa)

### DISCUSSION THREAD

**Discussion by jfruiz11375** [TPC-Council/HammerDB#411] [General]
Error when trying to run ./hammerdb getting

wish8.6: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by ./lib/libtk8.6.so)
wish8.6: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by ./lib/libtcl8.6.so)

**Comment by sm-shaw** [comment]
Yes this is known, HammerDB v4.5 was compiled against a newer version of GLIBC so will not run on Red Hat 8.0 or earlier but should on Red Hat 9.0 - there is an outstanding question for @abondvt89 about whether we want to provide a build of HammerDB v4.5 for older versions of Red Hat specifically or just update the documentation as the [test matrix](https://hammerdb.com/docs/ch01s04.html) is out of date? 

Note that HammerDB supports [building from source](https://hammerdb.com/docs/ch01s13.html) with a single command, and therefore it is possible for the user to now build for any earlier Linux versions (hence the reason to compile on a newer OS).

**Comment by jfruiz11375** [comment]
So should I download and run the 4.4 version?

**Reply by sm-shaw** [reply]
4.4 was compiled on Red Hat 7 so will run on both 7 & 8

**Comment by jfruiz11375** [comment]
We will go the RHEL 9 thanks for the info.

**Comment by sm-shaw** [comment]
A Red Hat 8 compatible [download](https://www.hammerdb.com/download.html. ) has been added to the release files as discussed and agreed in the TPC-OSS subcommittee due to the dependency of Oracle and Db2 on this version of RHEL.

**Comment by jfruiz11375** [comment]
Thanks a lot.

Javier Ruiz
Independent Database Consultant
Database Administration Services

“We cannot solve our problems with the same thinking we used to create them”

My sites

http://db12c.blogspot.com/

http://cloudcontrol12c.blogspot.com/
